### PR TITLE
Adding Keyword: parameter array

### LINKED ISSUE #12574: Params, Splat, every language has a different term for this
If there is a generic term for splats (Ruby)/params parameters (C#, and the most vague way of referring to this concept I can imagine), please include it in this documentation for better searchability as well as general knowledge.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: dbdb1f5e-a34b-6f30-2262-7c659e1ee466
* Version Independent ID: c985d741-8cf9-4486-d803-ddee5904faa1
* Content: [params keyword - C# Reference](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/params)
* Content Source: [docs/csharp/language-reference/keywords/params.md](https://github.com/dotnet/docs/blob/master/docs/csharp/language-reference/keywords/params.md)
* Product: **dotnet-csharp**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

### PR DESCRIPTION
**Pull request by AndrewCook**
## Summary

Adding parameter array keywords and text.

Fixes #12574 (if available)

### CODE REVIEW

File: docs/csharp/language-reference/keywords/params.md [hunk 1]
```diff
@@ -6,14 +6,16 @@ ms.date: 07/20/2015
 f1_keywords: 
   - "params_CSharpKeyword"
   - "params"
+  - "parameter array"
```
Comment (gewarren) [inline docs/csharp/language-reference/keywords/params.md]: I think this should probably be removed. It has to correlate to an actual keyword in the product.

File: docs/csharp/language-reference/keywords/params.md [hunk 2]
```diff
@@ -6,14 +6,16 @@ ms.date: 07/20/2015
...
 helpviewer_keywords: 
   - "parameters [C#], params"
   - "params keyword [C#]"
+  - "parameter array"
 ms.assetid: 1690815e-b52b-4967-8380-5780aff08012
 ---
 # params (C# Reference)
 
-By using the `params` keyword, you can specify a [method parameter](method-parameters.md) that takes a variable number of arguments.
+By using the `params` keyword, you can specify an array of [method parameter](method-parameters.md) that takes a variable number of arguments.
```
Comment (gewarren) [inline docs/csharp/language-reference/keywords/params.md]: This sentence no longer makes sense. I would leave it as is and instead add "parameter array" to the `title` metadata.

**Review by gewarren — DISMISSED** [review]

**Comment by BillWagner** [comment]
@AndrewCook Do you want to update this based on @gewarren 's comments? Once you do that, we'll :shipit:

File: docs/csharp/language-reference/keywords/params.md (hunk 2 above)
Comment (AndrewCook) [reply docs/csharp/language-reference/keywords/params.md]: @gewarren Are you suggesting changing the title from "params keyword - C# Reference" to "params keyword parameter array- C# Reference"

Comment (gewarren) [reply docs/csharp/language-reference/keywords/params.md]: I'd suggest "params keyword for parameter arrays - C# reference".

**Review by gewarren — COMMENTED** [review]
I left a couple comments. I think this first few paragraphs could use some other touch-ups, but that isn't directly related to the issue that's being fixed in this PR.

**Comment by gewarren** [comment]
@BillWagner Could you review the changes I made?

**Comment by gewarren** [comment]
Ping @billwagner.

**Review by BillWagner — APPROVED** [review]
Thanks for the updated changes @AndrewCook and @gewarren 

I'll :shipit: now.

**Pull request merged**